### ISSUE TITLE
Option to configure dynamically instead of environmental variables

### ISSUE THREAD

**Issue by jstdk** [WhatsApp/WhatsApp-Nodejs-SDK#32]
Happy to see an official SDK. I do not understand the /env route though, it will make the usage of this library impossible for projects that setup the webhooks dynamically. 

It would be great if there would an overwrite option in the constructor like all other SDK's have (Viber, Telegram, Discord, Line, etc)

**Comment by rahulX001** [comment]
One option for dynamically configuring an application is to use a configuration service that stores the application's configuration settings and provides an API for retrieving them at runtime. The application can then make API requests to the configuration service to retrieve the configuration settings it needs.

**Comment by jstdk** [comment]
@rahulX001, true, that would be nice, but it is not necessary to have a service for it to make it work. I have my own services to store bot information, the only thing I need is a way to set the webhooks through code. The only anomaly (next to WhatsApp/Messenger) would be Discord, where the webhook is also manually set in the  developer portal, yet they provide a function to fetch the webhook based on a token.

**Comment by sumit334** [comment]
Hello, I just to reach out to you @whatsapp  for some sort of issue in the whatsapp app(not web) , the program is that the whatsapp domain ,
wa.me/  have a some sort of bug in it if we use ," wa.me/settings " , if we send this link in the chat of any user or group in the whatsapp app , that group or contact can't be reached/opened by the any of the person. It would show that there is a bug in the content try to fix that bug first by clearing the cache , but even on clearing the cache that would not enough to fix the issue.

There is a solution to the issue, as it is not the issue on the whatsapp web , so we could simply use web to delete that message and get the contact or the group back, 
Actually the wa.me/settings block the chats, 
I would request you to please review the issue and find a way to fix the bug
Thank you
Regards Sumit

**Comment by jstdk** [comment]
@sumit334, you are adding a new issue to an existing issue. Please open a new issue for this

**Comment by sumit334** [comment]
Okay

**Comment by dmitryvinn-fb** [comment]
Closing - please read more in the [following issue](https://github.com/WhatsApp/WhatsApp-Nodejs-SDK/issues/31).
